### PR TITLE
IrfanView

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**

### CODE REVIEW

File: pending/xnview.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnview.new">
```
Comment (az0) [inline pending/xnview.xml]: File and ID conflict with https://github.com/az0/cleanerml/pull/145

File: pending/irfanview.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-20
+
+-->
+<cleaner id="irfanview">
```
Comment (az0) [inline pending/irfanview.xml]:
```suggestion
<cleaner id="irfanview" os="windows">
```

File: pending/irfanview.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  <option id="opendir">
+    <label>Open Directory</label>
+    <description>Directory shown in the open dialog</description>
+        <action command="ini" search="glob" path="%AppData%\IrfanView\i_view32.ini" name="OpenDir"/>
+        <action command="ini" search="glob" path="%AppData%\IrfanView\i_view64.ini" name="OpenDir"/>
+  </option>
+  <option id="mrud">
+    <label>Most recent used directory Thumbnails</label>
+    <description>Most recent used directory (MRUD) - Thumbnails</description>
+        <action command="ini" search="glob" path="%AppData%\IrfanView\i_view32.ini" name="MRUD*"/>
```
Comment (az0) [inline pending/irfanview.xml]: Wildcard current not supported for `command="ini"`

File: pending/xnview.xml (hunk 1 above)
Comment (Tobias-B-Besemer) [reply pending/xnview.xml]: OK, this should be solved...

File: pending/irfanview.xml (hunk 3 above)
Comment (Tobias-B-Besemer) [reply pending/irfanview.xml]: Solved.

**Comment by Tobias-B-Besemer** [comment]
OK, new version is there...

**Pull request merged**